dsf-fhir/dsf-fhir-server/src/test/java/org/highmed/dsf/fhir/profiles/OrganizationProfileTest.java

```diff
@@ -89,7 +89,7 @@ public void testOrganizationProfileNotValid2() throws Exception
 		result.getMessages().stream().map(m -> m.getLocationString() + " " + m.getLocationLine() + ":"
 				+ m.getLocationCol() + " - " + m.getSeverity() + ": " + m.getMessage()).forEach(logger::info);
 
-		assertEquals(1, result.getMessages().stream().filter(m -> ResultSeverityEnum.ERROR.equals(m.getSeverity())
+		assertEquals(2, result.getMessages().stream().filter(m -> ResultSeverityEnum.ERROR.equals(m.getSeverity())
```

Comment:
Why did it change from 1 to 2?

Reply:
Something changed in the HAPI validation code from 5.0.2 to 5.1.0. The testOrganizationProfileNotValid2 test case now produces two errors: Organization.type[0].coding[0].code missing and Organization.type[0].coding[0].code not being part of ttp://highmed.org/fhir/ValueSet/organization-type.